spec.html

```diff
@@ -903,21 +903,22 @@ <h1>Static Semantics</h1>
       <p>The Number value 0, alternatively written 0<sub>𝔽</sub>, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as *+0* for clarity.</p>
       <p>This specification denotes most numeric values in base 10; it also uses numeric values of the form 0x followed by digits 0-9 or A-F as base-16 values.</p>
-      <p>In certain contexts, an operation is specified which is generic between Numbers and mathematical values. In these cases, the subscript can be a variable; _t_ is often used for this purpose, for example 5<sub>_t_</sub> &times; 10<sub>_t_</sub> = 50<sub>_t_</sub> for any _t_ ranging over ℝ and 𝔽, since the values involved are within the range where the semantics coincide.</p>
-      <p>Conversions between mathematical values and numbers are never implicit, and always explicit in this document. A conversion from a mathematical value to a Number is denoted as "the Number value for _x_", and is defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>. A conversion from a Number to a mathematical value is denoted as "the <dfn id="mathematical-value">mathematical value</dfn> of _x_", or ℝ(_x_). Note that the mathematical value of non-finite values is not defined, and the mathematical value of *+0* and *-0* is the mathematical value 0<sub>ℝ</sub>.</p>
-      <p>When the term <dfn id="integer">integer</dfn> is used in this specification, it refers to a Number value whose mathematical value is in the set of integers, unless otherwise stated: when the term <dfn id="mathematical integer">mathematical integer</dfn> is used in this specification, it refers to a mathematical value which is in the set of integers. As shorthand, integer<sub>_t_</sub> can be used to refer to either of the two, as determined by _t_.</p>
+      <p>In certain contexts, an operation is specified which is generic between Numbers, BigInts and mathematical values. In these cases, the subscript can be a variable; _t_ is often used for this purpose, for example 5<sub>_t_</sub> &times; 10<sub>_t_</sub> = 50<sub>_t_</sub> for any _t_ ranging over ℝ, 𝔽 and ℤ, since the values involved are within the range where the semantics coincide.</p>
+      <p>Conversions between mathematical values and numbers are never implicit, and always explicit in this document. A conversion from a mathematical value to a Number is denoted as "the Number value for _x_" or 𝔽(_x_), and is defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>. A conversion from a Number to a mathematical value is denoted as "the <dfn id="mathematical-value">mathematical value</dfn> of _x_", or ℝ(_x_). Note that the mathematical value of non-finite values is not defined, and the mathematical value of *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub> is the mathematical value 0<sub>ℝ</sub>.</p>
```

Comment:
We capitalise "Number value. Are we also capitalising "Mathematical value"?

```suggestion
      <p>Conversions between Mathematical values and Numbers are never implicit, and always explicit in this document. A conversion from a Mathematical value to a Number is denoted as "the Number value for _x_" or 𝔽(_x_), and is defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>. A conversion from a Number to a Mathematical value is denoted as "the <dfn id="mathematical-value">Mathematical value</dfn> of _x_", or ℝ(_x_). Note that the Mathematical value of non-finite values is not defined, and the Mathematical value of *+0*<sub>𝔽</sub> and *-0*<sub>𝔽</sub> is the Mathematical value 0<sub>ℝ</sub>.</p>
```

Reply:
FWIW I think we should go the other direction. `Number` is capitalized because it is talking about an ES type rather than the usual meaning of the word. "mathematical" is just a normal adjective and should not be capitalized.

Reply:
I'm also fine with that. I care more that we are consistent.